release.md

```diff
@@ -1,67 +0,0 @@
-## [v1.8.1]
```

Comment:
This file seems to (by and large) duplicate the CHANGELOG.md which you also deleted. How/where do we keep a log of changes if we're discontinuing these two files?

Reply:
See comment above.. My view is git is our change log. And GitHub can automatically generate for you. See <https://docs.github.com/en/repositories/releasing-projects-on-github/automatically-generated-release-notes>. Otherwise we need to compile a change log and I don't think that is useful beyond announcements on Google Groups. Open to options for this.. Let me know your thoughts.

Reply:
Agreed. Let's not waste time on maintaining a document that duplicates github info.